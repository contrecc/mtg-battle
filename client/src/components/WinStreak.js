import React from 'react';

export default props => {
  const { winStreak } = props;
  return (
    <div>
      <h2 className="text-info" style={{ marginBottom: '40px' }}>
        Win Streak: {winStreak}
      </h2>
    </div>
  );
};
