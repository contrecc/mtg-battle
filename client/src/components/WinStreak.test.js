import WinStreak from './WinStreak';

describe('WinStreak', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<WinStreak />);
    expect(wrapper).toMatchSnapshot();
  });

  it('renders correctly with props', () => {
    const sentProps = 1;
    const wrapper = mount(<WinStreak winStreak={sentProps} />);
    const h2 = wrapper.find('h2');
    expect(h2.text()).toEqual(`Win Streak: ${sentProps}`);
  });
});
