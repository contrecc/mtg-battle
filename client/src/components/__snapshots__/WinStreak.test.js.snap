// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WinStreak renders correctly 1`] = `
<div>
  <h2
    className="text-info"
    style={
      Object {
        "marginBottom": "40px",
      }
    }
  >
    Win Streak: 
  </h2>
</div>
`;
